e2e/support/helpers/e2e-dashboard-helpers.js

```diff
@@ -172,8 +172,11 @@ export function goToTab(tabName) {
 }
 
 export function moveDashCardToTab({ dashcardIndex = 0, tabName }) {
-  getDashboardCard(dashcardIndex).realHover().icon("move_card").click();
-  menu().findByText(tabName).click();
+  getDashboardCard(dashcardIndex)
+    .realHover()
+    .icon("move_card")
+    .click({ force: true });
```

Comment:
Running tests locally doesn't give us ANY confidence they will pass in the CI.

Reply:
> force: true in second click also seems unnecessary

This was the sole reason behind all failures in CI.

Reply:
Now we're talking 🎉 
https://github.com/metabase/metabase/actions/runs/6783006614/job/18436382597

20/20 runs passing in CI.
That gives us confidence that this will not flake anymore.

Reply:
Yep it should open on hover, my bad for using click on the helper method, thanks for looking into it!

Reply:
> This was the sole reason behind all failures in CI.

It wasn't, it was unrelated.

The reason was that the menu was triggered with `hover` event, not `click`, and triggering `click` in cypress will not always trigger `hover` event.